src/rate/currencyconverterapi.ts

```diff
@@ -5,6 +5,10 @@ import {
   EdgeRatePlugin
 } from 'edge-core-js/types'
 
+const asInitOptions = asObject({
```

Comment:
Missed spookyswap and tombswap. Those plugins are buried real deep in the folder hierarchy for some reason.

Reply:
Added them.